metricbeat/module/system/diskio/_meta/docs.asciidoc

```diff
@@ -11,4 +11,17 @@ This metricset is available on:
 [float]
 === Configuration
 
-There are no configuration options for this metricset.
+*`diskio.include_devices`*:: When the `diskio` metricset is enabled, you can use the
+`diskio.include_devices` option to define a list of device names to pre-filter the
+devices that are reported. Filters only exact matches
+If not set, all disk devices are returned
```

Comment:
If not set or an empty array is set, all disk devices are returned. 

(also missing dot at the end).